LuckParser/Models/ParseModels/CombatReplay/CombatReplay.cs

```diff
@@ -25,6 +25,7 @@ public class CombatReplay
         // actors
         private readonly List<CircleActor> _circleActors = new List<CircleActor>();
         private readonly List<DoughnutActor> _doughnutActors = new List<DoughnutActor>();
+        private readonly List<RectangleActor> _rectangleActors = new List<RectangleActor>();
```

Comment:
same remark as the fields for rectangle actor (no need to touch the rest, I'll do it once I get back to CR stuff)